### PR TITLE
First cut of error message improvement: following PDS-45779

### PR DESCRIPTION
**Pull request by jboreiko**
I'm looking for ideas on how to improve this error message.  May be worth taking the time to enumerate the different scenarios we have seen this in since it seems to come up somewhat frequently (or at least provide a link to an portion of the OSS docs discussing this)

---
This change is [Reviewable](https://reviewable.io/reviews/palantir/atlasdb/981)

### CODE REVIEW

File: atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/timestamp/InDbTimestampBoundStore.java [hunk 1]
```diff
@@ -93,18 +93,18 @@ public Long run(Connection connection) throws SQLException {
...
+                                            + "to run in an HA configuration.");
                         }
                     } else {
                         // disappearance
                         throw new IllegalStateException(
                                 "Unable to retrieve a timestamp when expected. "
-                                + "You probably reseeded the database while the timestamp server was running. "
-                                + "This deloyment is in a dangerous state and you should take down "
-                                + "everything (all servers) and reseed the database again.");
+                                        + "This service is is in a dangerous state and should be taken down "
```
Comment (gsheasby) [inline atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/timestamp/InDbTimestampBoundStore.java]: "is is"

**Comment by gsheasby** [comment]
Yep, we could have a troubleshooting section of the docs which contains a paragraph about this issue.

File: atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/timestamp/InDbTimestampBoundStore.java [hunk 2]
```diff
@@ -93,18 +93,18 @@ public Long run(Connection connection) throws SQLException {
...
                             throw new MultipleRunningTimestampServiceError(
                                     "Timestamp limit changed underneath us (limit in memory: " + currentLimit
-                                    + ", limit in DB: " + oldLimit + "). This may indicate that "
-                                    + "another timestamp service is running against this database!  "
-                                    + "If you get this error check that your client.prefs in dispatch "
-                                    + "is configured correctly.");
+                                            + ", limit in DB: " + oldLimit + "). This may indicate that "
+                                            + "another timestamp service is running against this database!  "
+                                            + "This may indicate that your services are not properly configured "
+                                            + "to run in an HA configuration.");
```
Comment (tpetracca) [inline atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/timestamp/InDbTimestampBoundStore.java]: Or have CLI/Console run against them (or something to that affect)

**Comment by gsheasby** [comment]
Thumbs-up acquired from @tpetracca via chat...

**Pull request merged**